### ISSUE TITLE
 WAI-ARIA group is missing a name

### ISSUE THREAD

**Issue by mhk33** [sul-dlss/blacklight-hierarchy#35]
We run our catalog through SiteImprove to check accessibility and get the following WCAG 2.0 Level A error:

> 
> A group created with the WAI-ARIA role="group" or role="radiogroup" is missing a name. 
> How to fix it: Use either the WAI-ARIA attribute 'aria-label' or 'aria-labelledby' to name the group.

It is referring to each use of this list in the blacklight-hierarchy gem:

`<ul role="group" style="display: none;">`

It looks like the <ul> needs the aria-label or aria-labelledny attribute.
